.env

```diff
@@ -0,0 +1,4 @@
+REPLICATE_API_TOKEN=
+NEXT_PUBLIC_UPLOAD_IO_API_KEY=public_12a1yKf3LPX7yGpPqZTYArTL7va5
+UPSTASH_REDIS_REST_URL=
+UPSTASH_REDIS_REST_TOKEN=
```

Comment:
Perfect: the spinner looks great. I have just one request:

Please remove the .env file from the PR. 

💡Tips: with Next.js you can use .env.local for your env variables and it will get ignored by default.

If you need any help then feel free to reach me here or on discord